### ISSUE TITLE
Increase the time during which a "reset password" link is working

### ISSUE THREAD

**Issue by shenriod** [k-box/k-box#175]
### Short Description of the issue

I reset the password of a user. However, when she tried to login (1 day later), the system told her that the link is only valid 5 minutes. I think that 24 or 48 hours should be a more realistic value

![screen](https://user-images.githubusercontent.com/23653367/48475013-a8e3d800-e81d-11e8-8ab8-441703b1a85c.png)

### Expected behavior

Extend the validity period of a password reset link

### General information

Version of your K-Box: 0.23

URL of the page where the problem has occured:

Type and version of your browser: 

In what language are you using your K-Box?

**Comment by shenriod** [comment]
I guess this makes sense when the user him / herself requests a new password ("Forgot password" on the Login page).

But when the Administrator generates a new password, then the 5 minutes limit is not realistic, since the admin is not necessarily in live communication with the user. Maybe he has just received an email saying "I don't know my password, please help".

I guess, in this case, the optimal behavior would be to write to the user "please go to the login page and request a new password when you are ready to do so". But for "low-tech" users it is sometimes easier / quicker for the admin to generate the new password himself

**Comment by avvertix** [comment]
> Maybe he has just received an email saying "I don't know my password, please help".
> I guess, in this case, the optimal behavior would be to write to the user "please go to the login page and request a new password when you are ready to do so". But for "low-tech" users it is sometimes easier / quicker for the admin to generate the new password himself

Yes, if you receive such support request, you should explain the user how he/she can reset the password.
